scss/navbar.scss

```diff
@@ -0,0 +1,26 @@
+.navbar {
+    overflow: hidden;
+    background-color: white;
+    border-bottom: 1px solid black;
+    position: fixed; 
+    top: 0; 
+    height: $nav-bar-height;
+    width: 100%;
+    z-index: 9999;
+}
+
+.navbar .logo {
+    padding: 30px 20px;
+    height: $nav-bar-height;
+    width: auto;
+}
+
+.navbar .right-title {
+    font-family: $font-stack;
+    font-size: $text-big;
+    float: right;
```

Comment:
this is a really old practice that was used a lot in 2012. The problem with this is that floated elements stop taking space (like absolute positioned elements). this problem is easily solved by setting flex and justify content space between to the header.